ibis/pandas/udf.py

```diff
@@ -604,3 +526,73 @@ def wrapped(*args):
             return wrapped
 
         return wrapper
+
+
+@pre_execute.register(ops.ElementWiseVectorizedUDF)
+@pre_execute.register(ops.ElementWiseVectorizedUDF, PandasClient)
+def pre_execute_elementwise_udf(
+    op, *clients, scope=None, aggcontet=None, **kwargs
```

Comment:
should consider making pandas/udf/ a sub-directory as more things are added, but ok for now.

Reply:
Duly noted. I will incorporate this into future changes for other types of UDFs.